pymongo/auth_aws.py

```diff
@@ -35,6 +27,8 @@
     from pymongo.auth import MongoCredential
     from pymongo.pool import Connection
 
+_HAVE_MONGODB_AWS = importlib.util.find_spec("pymongo_auth_aws") is not None
```

Comment:
Given the problems we've seen, I'm a little wary of using importlib at all. How about we remove _HAVE_MONGODB_AWS instead? Looks like it's only used where we do the import anyway.

Reply:
Done